roles/edpm_network_config/tasks/main.yml

```diff
@@ -36,6 +36,9 @@
         name: NetworkManager
         state: restarted
       when: nm_ovs_status.changed
+    - name: Render network_config variable
+      ansible.builtin.set_fact:
+        network_config: "{{ lookup('template', edpm_network_config_nmstate_template ) }}"
```

Comment:
Where is this var used? Isn't `network_state` the var used by linux-system-reoles[1]? 

[1] https://github.com/linux-system-roles/network#variables

Reply:
indeed, i thought `network_state` and wrote `network_config`. :shrug: 

I'm scared that the tests passed anyway.